kbi/0008/index.rst

```diff
@@ -0,0 +1,65 @@
+.. index::
+   single: <annex>; <remove a git annex>
+
+KBI0008 Remove a git annex
+==========================
```

Comment:
```suggestion
KBI0008: Remove a dataset's annex
=============================
```

To match the style of other KBIs (the colon), and to disambiguate the location (annex) from the software (git annex)